lib/full_text_search/similar_searcher.rb

```diff
@@ -60,6 +71,13 @@ def after_destroy(record)
                                record.id,
                                "destroy",
                                issue_id: record.journalized_id)
+            when Attachment
+              return unless record.container_type == Issue.name
+              FullTextSearch::UpdateIssueContentJob
+                .perform_later(record.class.name,
+                               record.id,
+                               "destroy",
+                               issue_id: record.container_id)
```

Comment:
Ah, can we avoid reusing `FullTextSearch::SimilarSearch::Model` for `Journal` and `Attachment`?

We just need `after_commit`/`after_destroy` for `Journal` and `Attachment`. (They don't need `FullTextSearch::SimilarSearch::Model::InstanceMethods`.)

Reply:
fix: [9a67176](https://github.com/clear-code/redmine_full_text_search/pull/167/commits/9a671760f9a886711b960f75072dabc132c18eed) Sure, I extract `after_commit`/`after_destroy` as a `FullTextSearch::IssueContentSynchronizable`.